index.html

```diff
@@ -1,258 +1,388 @@
 <!doctype html>
 <html lang="en">
-  <link rel="icon" href="./images/logo_notebook.png" />
+  <link rel="icon" href="./images/favicon.png" />
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Automation Quality Assurance Engineer</title>
     <link rel="stylesheet" href="style.css" />
   </head>
   <body class="page-portfolio">
-    <header>
-      <img
-        class="page-portfolio-header-image"
-        src="./images/portrait.png"
-        alt="broken image portrait"
-      />
-      <h1 class="page-portfolio-header">
-        Hello! <br />
-        I'm <em class="page-portfolio-header-name"> Yuliia </em><br />
-        Automation QA Engineer & <br />
-        Full-stack Developer
-      </h1>
+    <nav class="navbar">
+      <div class="navbar__brand">
+        <script
+          src="https://unpkg.com/@dotlottie/player-component@latest/dist/dotlottie-player.mjs"
+          type="module"
+        ></script>
+
+        <dotlottie-player
+          src="https://lottie.host/6c321171-97be-42a2-8ce6-09624a820043/CyPgEsSz55.json"
+          background="transparent"
+          speed="1"
+          style="width: 100px; height: 100px"
+          loop
+          autoplay
+        ></dotlottie-player>
+      </div>
+      <div class="navbar__menu">
+        <a class="navbar__menu-link" href="#home-page">Home</a>
+        <a class="navbar__menu-link" href="#skill-info">Skills</a>
+        <a class="navbar__menu-link" href="#form-info">Form</a>
+        <a class="navbar__menu-link" href="#contact-info">Contact</a>
+      </div>
+    </nav>
 
-      <p class="header-text">
-        I always ready for <em>new challenges</em>. <br />
-        <em>Education </em> is my <em>passport </em>to the future, <br />for
-        tomorrow belongs only to the people <br />
-        who prepare for it today.
-      </p>
+    <header>
+      <div class="header" id="home-page">
+        <div class="header__brand">
+          <div class="header__brand__image">
+            <script
+              src="https://unpkg.com/@dotlottie/player-component@latest/dist/dotlottie-player.mjs"
+              type="module"
+            ></script>
+            <dotlottie-player
+              src="https://lottie.host/ee703296-a8a9-419f-90dd-6fd36499b0a2/oWuEBCSI2J.json"
+              background="transparent"
+              speed="1"
+              style="width: 300px; height: 300px"
+              loop
+              autoplay
+            ></dotlottie-player>
+          </div>
+          <div class="header__brand__info">
+            <h1 class="header__brand__info-title">
+              Hello! <br />
```

Comment:
We should start moving away from <br>.
You can achieve the same result by adding multiple p tags to do a line break.